unit-test-coverage/pc-rtems/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+######################################################################
+#
+# CMake build recipe for pc-rtems PSP white-box coverage tests
+#
+######################################################################
+
+include_directories(${CFEPSP_SOURCE_DIR}/fsw/pc-rtems/inc)
+include_directories(${PSPCOVERAGE_SOURCE_DIR}/shared/inc)
+
+# Target names use a "ut" prefix to avoid confusion with the FSW targets
+set(CFE_PSP_TARGETNAME "ut-${SETNAME}")
+add_subdirectory(${CFEPSP_SOURCE_DIR}/fsw/${SETNAME} ${CFE_PSP_TARGETNAME}-impl)
+add_subdirectory(${CFEPSP_SOURCE_DIR}/fsw/shared ${CFE_PSP_TARGETNAME}-shared)
+
+# The UT assert library defines OS_Application_Startup, so this redefines ours with a "UT_" prefix
+target_compile_definitions(psp-${CFE_PSP_TARGETNAME}-impl PRIVATE
+    OS_Application_Startup=UT_OS_Application_Startup
+)
+
+foreach(TGT psp-${CFE_PSP_TARGETNAME}-impl psp-${CFE_PSP_TARGETNAME}-shared)
+    target_compile_options(${TGT} PRIVATE ${UT_COVERAGE_COMPILE_FLAGS})
+    target_include_directories(${TGT} BEFORE PRIVATE ${PSPCOVERAGE_SOURCE_DIR}/ut-stubs/override_inc)
+endforeach()
```

Comment:
Should be able to remove `psp-${CFE_PSP_TARGETNAME}-shared` which makes the `foreach` unnecessary.

Reply:
Retracted comment due to dependencies on shared APIs